src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -226,7 +202,7 @@ def _run_app_internal(self, app_id, params, tag, version, cell_id, run_id, **kwa
         self.spec_manager.check_app(app_id, tag, raise_exception=True)
 
         if version is not None and tag != "release":
-            if re.match(version, '\d+\.\d+\.\d+') is not None:
+            if re.match(r'\d+\.\d+\.\d+', version) is not None:
                 raise ValueError("Semantic versions only apply to released app modules. You can use a Git commit hash instead to specify a version.")
```

Comment:
I'm not sure... but that's all over the place in appmanager.py (and in KBase in general). I think @sychan wants to adopt the < 80 char limit.